packages/ui/src/domains/LandingPageBuilder/LadingPageBuilder/components/selectFont.tsx

```
@@ -0,0 +1,30 @@
+	SelectTrigger,
+	SelectValue,
+	SelectContent,
+	SelectItem,
+} from "@repo/ui/components/ui/select";
+
+export default function SelectFont({
+	font,
+	onUpdate,
+}: {
+	font: string;
+	onUpdate: (font: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)